scripts/scil_visualize_bingham_fit.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Visualize 2-dimensional Bingham volume slice loaded from disk. The volume is
+assumed to be saved from scil_compute_bingham_fodf_metrics.py.
+
+Given an image of Bingham coefficients, this script displays a slice in a
+given orientation.
+"""
+
+import argparse
+
+import nibabel as nib
+import numpy as np
+
+from dipy.data import get_sphere
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+from scilpy.viz.scene_utils import (create_bingham_slicer,
+                                    create_scene, render_scene)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__,
+                                formatter_class=argparse.RawTextHelpFormatter)
+
+    # Positional arguments
+    p.add_argument('in_bingham', help='Input SH image file.')
+
+    # Window configuration options
+    p.add_argument('--slice_index', type=int,
+                   help='Index of the slice to visualize along a given axis. '
+                        'Defaults to middle of volume.')
+
+    p.add_argument('--win_dims', nargs=2, metavar=('WIDTH', 'HEIGHT'),
+                   default=(768, 768), type=int,
+                   help='The dimensions for the vtk window. [%(default)s]')
+
+    p.add_argument('--interactor', default='trackball',
+                   choices={'image', 'trackball'},
+                   help='Specify interactor mode for vtk window. '
+                        '[%(default)s]')
+
+    p.add_argument('--axis_name', default='axial', type=str,
+                   choices={'axial', 'coronal', 'sagittal'},
+                   help='Name of the axis to visualize. [%(default)s]')
+
+    p.add_argument('--silent', action='store_true',
+                   help='Disable interactive visualization.')
+
+    p.add_argument('--output', help='Path to output file.')
+
+    add_overwrite_arg(p)
+
+    sphere_choices = {'symmetric362', 'symmetric642', 'symmetric724',
```

Comment:
Why a dictonnary instead of an array.

Reply:
It would seem I was declaring a c++ array... Should (and will) be a list.

Reply:
Done here 60364e4